scripts/automation/pin_and_tag_artifacts.sh

```diff
@@ -119,17 +119,17 @@ function pin_and_tag_artifacts_GridEditorPlugin() {
     # pin the GridEditorPlugin msi
     local grideditorplugin_msi_build_number=$(
         python ${scripts_path}/get_build_number.py \
-            --build_config_id GridEditor_GridEditorPlugin${forked_repo_suffix}_Deliverables_StandAloneInstaller \
+            --build_config_id GridEditor_GridEditorPlugin${forked_repo_suffix}_Deliverables_Installers_DGridEditorSignedMsiSInstallers \
             --version ${version} \
             --last_successful_build \
             --teamcity_access_token ${teamcity_access_token}
     )
-    local msi_file_name="GridEditor ${dhydro_suite_version} Stand-alone (${grideditorplugin_msi_build_number}).msi"
+    local msi_file_name="D-GridEditor ${dhydro_suite_version} (${grideditorplugin_msi_build_number}).msi"
     python ${scripts_path}/pin_artifact.py \
         --branch_name ${release_branch} \
-        --build_config_id GridEditor_GridEditorPlugin${forked_repo_suffix}_Deliverables_StandAloneInstaller \
+        --build_config_id GridEditor_GridEditorPlugin${forked_repo_suffix}_Deliverables_Installers_DGridEditorSignedMsiSInstallers \
         --tag ${tag} \
-        --artifact_path "setup/GridEditor/bin/Release/stand-alone" \
+        --artifact_path "installer/setup/GridEditor/bin/Release/stand-alone" \
```

Comment:
is the artifact path correct?
![image](https://github.com/user-attachments/assets/637279ef-943e-4bf4-9a4e-a6a58a983f8c)